offender-search-prod/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: offender-search-prod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
```

Comment:
This namespace is called "prod", but you've got "is-production" "false" and "environment-name" "dev" - is that correct?

Reply:
thanks - will address